fe/fe-core/src/main/java/com/starrocks/sql/optimizer/task/EnforceAndCostTask.java

```diff
@@ -408,8 +414,9 @@ private boolean canGenerateOneStageAgg(GroupExpression childBestExpr) {
         return true;
     }
 
-    private boolean computeCurrentGroupStatistics() {
+    private boolean checkCurrentGroupStatistics() {
         if (groupExpression.getInputs().stream().anyMatch(group -> group.getStatistics() == null)) {
+            Preconditions.checkState(false);
```

Comment:
why add this check?

Reply:
I think we never hit the branch, we can remove the check later